exporter/debugexporter/README.md

```diff
@@ -34,6 +34,70 @@ exporters:
     sampling_thereafter: 200
 ```
 
+## Verbosity levels
+
+The following subsections describe the output from the exporter depending on the configured verbosity level - `basic`, `normal` and `detailed`.
+The default verbosity level is `normal`.
+
+### Basic verbosity
+
+With `verbosity: basic`, the exporter outputs a single-line summary of received data with a total count of telemetry records for every batch of received logs, metrics or traces.
+
+Here's an example output:
+
+```console
+2023-11-10T22:49:03.510-0600    info    TracesExporter  {"kind": "exporter", "data_type": "traces", "name": "debug", "resource spans": 1, "spans": 2}
+```
+
+### Normal verbosity
```

Comment:
Since this is the default, I would prefer the documentation for `basic` to be under `normal` with `basic` referencing the `normal` level.

Reply:
Done. Although I believe this description fits better into the `basic` section and it's the `normal` behavior that is somehow incorrect and subject to be changed. I also beileve the default verbosity also needs to be changed from `normal` to `basic`. We can talk about this in future pull requests/issues though. :slightly_smiling_face:

Reply:
Thanks @astencel-sumo, i've opened up https://github.com/open-telemetry/opentelemetry-collector/issues/8844 to follow up on this comment